readme.md

```diff
@@ -2,40 +2,38 @@
 
 > Get the icon of a file or app as a PNG image *(macOS)*
 
-
 ## Install
 
-```
+```bash
 $ npm install --global file-icon-cli
 ```
 
-
 ## Usage
 
-```
+```bash
 $ file-icon --help
 
   Usage
     $ file-icon <app-name|bundle-id|file-path>
+    $ file-icon <app-name|bundle-id|file-path> > icon.png
 
   Options
     --size  Size of the icon [Max: 1024]
 
   Examples
     $ file-icon Safari
     /tmp/86ca9400-9f34-4a64-ab24-027d80f88b46/icon.png
+    $ file-icon Safari > icon.png
```

Comment:
Move this to the bottom instead. Looks better with it not having any logged output.